packages/core/contracts/UnirepSocial.sol

```diff
@@ -46,15 +46,15 @@ contract UnirepSocial {
     event PostSubmitted(
         uint256 indexed _epoch,
         uint256 indexed _epochKey,
-        string _postContent,
+        bytes32 _postContent,
```

Comment:
we should change these args to be something like `_contentHash` just to mark these as being hashes and not full content.